ego/premain/core/core.go

```diff
@@ -51,7 +51,7 @@ func PreMain(payload string, mounter Mounter, fs afero.Fs, hostEnviron []string)
 	// Convert host environment string array to key-value map
 	hostEnvironMap := make(map[string]string, len(hostEnviron))
 	for _, envVar := range hostEnviron {
-		splitString := strings.Split(envVar, "=")
+		splitString := strings.SplitN(envVar, "=", 2)
 		hostEnvironMap[splitString[0]] = splitString[1]
```

Comment:
```suggestion
		envName, envValue, _ := strings.Cut(envVar, "=")
		hostEnvironMap[envName] = envValue
```
The Go [docs recommend using Cut](https://pkg.go.dev/strings#SplitN) if splitting around the first instance of a separator.
There shouldn't be any functional difference though